crates/uv-python/src/managed.rs

```diff
@@ -250,7 +250,10 @@ impl ManagedPythonInstallations {
             .find_all()?
             .filter(move |installation| {
                 installation.key.os == os
-                    && installation.key.arch.family == arch.family
+                    && (arch.supports(installation.key.arch)
+                        // TODO(zanieb): Allow inequal variants, as `Arch::supports` does not
+                        // implement this yet. See https://github.com/astral-sh/uv/pull/9788
+                        || arch.family == installation.key.arch.family)
```

Comment:
Just retaining the hack from https://github.com/astral-sh/uv/pull/10722 for now

See https://github.com/astral-sh/uv/pull/9788